src/main/features/dashboard/index.ts

```diff
@@ -34,6 +35,28 @@ function render (claim: Claim, type: string): string {
   }
 }
 
+function formatAddress (address: Address): string {
```

Comment:
could have written using 'for in' or Object.keys

Reply:
if the sequence of key changes for some reason, address may start with zip code or city name instead of street name. 
also if any new key gets introduced which ideally should not, it may cause problem. this way we have better control, I think.